server/models/game.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+module.exports = function(sequelize, DataTypes) {
+  var Game = sequelize.define('Game', {
+    game_number: DataTypes.INTEGER,
```

Comment:
let's just clear out everything but the MVP stuff. Keep the first version simple.